src/pybind/mgr/orchestrator/module.py

```diff
@@ -1136,6 +1136,47 @@ def sort_by_field(d: DaemonDescription) -> Any:
 
             return HandleCommandResult(stdout=table.get_string())
 
+    @_cli_read_command('orch cert-store cert ls')
+    def _cert_store_cert_ls(self, format: Format = Format.plain) -> HandleCommandResult:
+        completion = self.cert_store_cert_ls()
+        cert_ls = raise_if_exception(completion)
+        if format != Format.plain:
+            return HandleCommandResult(stdout=to_format(cert_ls, format, many=False, cls=None))
+        else:
+            result_str = ''
+            # TODO: the nested if-else cases here look similar
+            # can this be simplified?
+            for k in cert_ls.keys():
+                if not isinstance(cert_ls[k], dict):
+                    result_str += f'{k} - {cert_ls[k]}\n'
+                else:
+                    result_str += f'{k}\n'
+                    for k2 in cert_ls[k].keys():
+                        if not isinstance(cert_ls[k][k2], dict):
+                            result_str += f'  {k2} - {cert_ls[k][k2]}\n'
```

Comment:
what about something like?

**Note**: I didn't test the code just as an idea on how can we use resistivity to introduce multi-level indentation. 

```
def process_dict(d, level=0):
    result_str = ""
    indent = "  " * level
    for k, v in d.items():
        if isinstance(v, dict):
            result_str += f'{indent}{k}\n'
            result_str += process_dict(v, level + 1)
        else:
            result_str += f'{indent}{k} - {v}\n'
    return result_str
```